salt/modules/syslog_ng.py

```diff
@@ -558,9 +539,7 @@ def _is_junction(arg):
     Return True, if arg is a junction statement.
     """
     return (
-        isinstance(arg, dict)
-        and len(arg) == 1
-        and next(six.iterkeys(arg)) == "junction"
+        isinstance(arg, dict) and len(arg) == 1 and next(iter(arg.keys())) == "junction"
```

Comment:
**question**: I don't know if it's worth holding up this PR, but.... 

```
isinstance(arg, dict) and "junction" in arg
```
seems better. Presumably a simple `"junction" in arg` wouldn't quite be strict enough, since list/str/tuple, etc could behave the same.

Reply:
Yeah, I'd prefer not to change this just in case it introduces a bug. I don't see anything about junction in the tests for this module :(